apps/modernization-ui/src/apps/page-builder/page/management/preview/PageDetails/PageDetails.tsx

```diff
@@ -0,0 +1,125 @@
+import {
+    Concept,
+    Condition,
+    PageInformation,
+    PageInformationChangeRequest,
+    PageInformationService
+} from 'apps/page-builder/generated';
+import { fetchConditions } from 'apps/page-builder/services/conditionAPI';
+import { fetchMMGOptions } from 'apps/page-builder/services/valueSetAPI';
+import React, { useEffect, useState } from 'react';
+import { Button, Form, Icon } from '@trussworks/react-uswds';
+import './PageDetails.scss';
+import { PageDetailsField } from './PageDetailsField';
+import { useNavigate, useParams } from 'react-router-dom';
+import { authorization } from 'authorization';
+import { PagesBreadcrumb } from '../../../../components/PagesBreadcrumb/PagesBreadcrumb';
+import { useForm } from 'react-hook-form';
+import { useAlert } from '../../../../../../alert';
+import { useGetPageDetails } from '../../useGetPageDetails';
+
+export const PageDetails = () => {
+    const token = authorization();
+    const { pageId } = useParams();
+    const [pageEvent, setPageEvent] = useState('');
+    const [conditions, setConditions] = useState<Condition[]>([]);
+    const [mmgs, setMmgs] = useState<Concept[]>([]);
+    const navigate = useNavigate();
+    const { alertError } = useAlert();
+    const { page } = useGetPageDetails();
+    const isEnabled = ['Initial draft', 'Published with draft', 'Draft'].includes(page?.status!);
+
+    const form = useForm<PageInformationChangeRequest>({
+        mode: 'onBlur',
+        defaultValues: {}
+    });
+    const { control } = form;
+    useEffect(() => {
+        fetchMMGOptions(token)
+            .then((data) => {
+                setMmgs(data);
+            })
+            .catch((error: any) => {
+                console.log('Error', error);
+            });
+        fetchConditions(token).then((data) => {
+            setConditions(data);
+        });
+    }, []);
+    useEffect(() => {
+        if (pageId) {
+            const getConditions = (data: any) => {
+                return data.map((dt: any) => dt.value);
+            };
+            PageInformationService.find({
+                authorization: authorization(),
+                page: Number(pageId)
+            }).then((data: PageInformation) => {
+                setPageEvent(data?.eventType?.value!);
+                const condition = getConditions(data?.conditions);
+                form.setValue('datamart', data?.datamart!);
+                form.setValue('messageMappingGuide', data?.messageMappingGuide?.value);
+                form.setValue('name', data?.name!);
+                form.setValue('description', data?.description!);
+                form.setValue('conditions', condition);
+            });
+        }
+    }, [pageId]);
+    const handleCancel = () => {
+        navigate(-1);
```

Comment:
Issue: using `navigate(-1)` will return the user to wherever they came to this page from. 99% of the time this will work as intended but if a user directly navigated here, this will not function as expected.

Given the route to this page is `pages/{pageId}/details`, you can use `navigate('..')` to route to `pages/{pageId}`